### PR TITLE
Update to the new API, updated servers and added movie server type

### PR DESCRIPTION
**Pull request by ahmed605**
Changes:

- Updating to the new Akinator API (as the old (used) one no longer works so the library wasn't working)

- Updating servers' URLs

- Adding movie server type (with tests)

### CODE REVIEW

File: Akinator.Api.Net.Tests/ClientIntegrationTests.cs [hunk 1]
```diff
@@ -1,561 +1,614 @@
...
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using System;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Akinator.Api.Net.Tests
-{
-    [TestClass]
-    public class ClientIntegrationTests
```
Comment (janniksam) [inline Akinator.Api.Net.Tests/ClientIntegrationTests.cs:12]: Can you explain what exactly you did change in this file? Did you only add the new movie tests or did something else change? Unfortunately I cannot view the changes with the github comparison tool :(

**Review by janniksam — COMMENTED** [review]
Can you edit the Readme aswell?

File: Akinator.Api.Net.Tests/ClientIntegrationTests.cs [hunk 2]
```diff
@@ -504,19 +558,6 @@ public async Task UndoWorksAsExpected()
             }
         }
 
-
-        [TestMethod]
```
Comment (janniksam) [inline Akinator.Api.Net.Tests/ClientIntegrationTests.cs:508]: Was this feature removed?

Comment (janniksam) [reply Akinator.Api.Net.Tests/ClientIntegrationTests.cs:508]: Why did you delete it?

File: Akinator.Api.Net.Tests/ClientIntegrationTests.cs (hunk 1 above)
Comment (ahmed605) [reply Akinator.Api.Net.Tests/ClientIntegrationTests.cs:12]: I only added movie tests, it looks like the GitHub diff viewer is acting up

File: Akinator.Api.Net.Tests/ClientIntegrationTests.cs (hunk 2 above)
Comment (janniksam) [reply Akinator.Api.Net.Tests/ClientIntegrationTests.cs:508]: Wait, I'm not sure anymore, did you really delete it?

Comment (ahmed605) [reply Akinator.Api.Net.Tests/ClientIntegrationTests.cs:508]: No, I just forgot to pull the latest changes so I added it again in the next commit

**Comment by janniksam** [comment]
Can you edit the Readme aswell?

File: Akinator.Api.Net.Tests/ClientIntegrationTests.cs [hunk 3]
```diff
@@ -118,7 +118,34 @@ public async Task SimpleWorkflowTest_English_Person()
                 }
             }
         }
-        
+
+        [TestMethod]
+        public async Task SimpleWorkflowTest_English_Movie()
```
Comment (janniksam) [inline Akinator.Api.Net.Tests/ClientIntegrationTests.cs]: Can you edit the Readme aswell?

**Review by janniksam — APPROVED** [review]
Thank you, everything good now 👍

**Comment by ahmed605** [comment]
@janniksam I have edited the README

**Pull request merged**